src/main/webapp/components/query-editor/query-editor.tsx

```diff
@@ -174,12 +162,7 @@ export default (props: EditorProps) => {
           marginTop: '10px',
         }}
       >
-        <Footer
```

Comment:
Discussed with UX, and the plan is to remove manual saving here in favor of saving on search.